xml/obs_package_formats.xml

```diff
@@ -278,8 +273,45 @@ systemctl enable sshd</screen>
   </example>
  </sect1>
 
- <sect1>
+ <sect1 xml:id="sec.obs.pkgformats.appimage">
   <title>&obs.ai;</title>
   <remark>toms 2017-08-18: TODO What is it and what is needed</remark>
+  <para>
+   An &obs.ai; is a bundled application including libraries that
+   allows to run on a variety of different distributions without
+   further installation or modification. The &obs.ai; contains all
+   libraries that are not already part of the base system.
+   &obs.ai; keeps the underlying operating system untouched, which means,
+   you do not need root permission nor installation.
+
+  </para>
+  <para> For building an image in &obs.ai; you need:</para>
+  <variablelist>
+   <varlistentry>
+    <term><filename>PACKAGE</filename></term>
+    <listitem>
+     <para>
+      The original, untouched source archive.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><filename><replaceable>PACKAGE</replaceable>.rpm</filename></term>
+    <listitem>
+     <para>
+      The successfully built RPM package.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><filename>appimage.yml</filename></term>
+    <listitem>
+     <para>
+      A YAML file which contains a list of dependencies and installation
```

Comment:
Some good source that explains YAML syntax here? Maybe the spec for appimage.yml in particular?

Reply:
I thought of that too, but obviously forgot to add this. Good point! 👍 
Probably http://yaml.org/ should be enough...

Reply:
Added `<link xlink:href="http://yaml.org">YAML</link>`